src/Maui/Features/Combat/Manage.razor.cs

```diff
@@ -83,6 +83,11 @@ public void Dispose()
         _cts.Dispose();
     }
 
+    public void RemoveFromCombat(InitiatedCreature initiatedCreature)
+    {
+        _initiatedCreatures.Remove(initiatedCreature);
```

Comment:
FYI; an error is thrown when you remove a creature that was currently active in initiative and then attempt to click end turn. Can either fix it here or in a separate issue. Might need some sort of check to make sure _activeCreature is not null before grabbing the next value.

Reply:
I'll fix this now 👍

Reply:
if the removed creature was the active creature, we need to selected the next active creature. if there are no active creatures left, it should end combat.